docs/users.rst

```diff
@@ -398,11 +398,17 @@ Dockerfile, the tags used within the repository are:
 
 - a unique tag including the timestamp (this tag is the only tag
   applied for scratch builds)
-- ``latest``
+- ``{version}-{release}`` (the ``version`` and ``release`` labels
+  together)
+
+If the ``tags`` value is set in container.yaml, those tags are applied
+to the image as floating tags.
+
+If ``tags`` is not used in container.yaml, the following tags are
+applied:
 - ``{version}`` (the ``version`` label)
-- ``{version}-{release}`` (the ``version`` and ``release`` labels together)
-- any additional tags configured in the git repository (named in the
-  ``additional-tags`` file)
+- ``latest``
```

Comment:
This is not showing a list with bullets but inline on single line in GH view. (GH issue?)
IMHO it needs extra newline after `applied:`

Reply:
Thanks, should be fixed now.